test/tax-lot/tax-lot.repository.mock.ts

```diff
@@ -2,22 +2,56 @@ import { generateMock } from "@anatine/zod-mock";
 import {
   findByBblRepoSchema,
   findByBblSpatialRepoSchema,
+  findZoningDistrictByTaxLotBblRepoSchema,
+  checkTaxLotByBblRepoSchema,
 } from "src/tax-lot/tax-lot.repository.schema";
 
 export class TaxLotRepositoryMock {
-  findByBblMocks = Array.from(Array(10), () =>
-    generateMock(findByBblRepoSchema, { seed: 1 }),
+  checkTaxLotByBblMocks = [
+    generateMock(checkTaxLotByBblRepoSchema, { seed: 1 }),
+  ];
+
+  async checkTaxLotByBbl(bbl: string) {
+    return this.checkTaxLotByBblMocks.find((row) => row.bbl === bbl);
+  }
+
+  findByBblMocks = Array.from(this.checkTaxLotByBblMocks, () =>
+    generateMock(findByBblRepoSchema, {
+      stringMap: {
+        bbl: () => this.checkTaxLotByBblMocks[0].bbl,
+      },
+    }),
   );
 
   async findByBbl(bbl: string) {
     return this.findByBblMocks.find((row) => row.bbl === bbl);
   }
 
-  findByBblSpatialMocks = Array.from(Array(10), () =>
-    generateMock(findByBblSpatialRepoSchema, { seed: 1 }),
+  findByBblSpatialMocks = Array.from(this.checkTaxLotByBblMocks, () =>
+    generateMock(findByBblSpatialRepoSchema, {
+      stringMap: {
+        bbl: () => this.findByBblMocks[0].bbl,
+      },
+    }),
   );
 
   async findByBblSpatial(bbl: string) {
     return this.findByBblSpatialMocks.find((row) => row.bbl === bbl);
   }
+
+  findZoningDistrictByTaxLotBblMocks = Array.from(
```

Comment:
I'm seeing this has the same functionality as using [array].map (it accesses the data from the generating array). As a preference, I think it's more idiomatic to use map. I also think it help readability to use it consistently. So, I would prefer we use `map` here, too.

But, this a preference. If you want to keep `Array.from` and refactor the other functions in such a way that they use also use it, I'm fine with that. The two big priorities are 1) getting rid of the hard-coded 0s and 2) being consistent